ports/raspberrypi/boards/keebio_elite_pi/mpconfigboard.mk

```diff
@@ -0,0 +1,9 @@
+USB_VID = 0xCB10
```

Comment:
Where did you get this USB PID? Most RP2040 PIDs come from raspberry pi: https://github.com/raspberrypi/usb-pid

Reply:
I got it from here:

https://github.com/qmk/qmk_firmware/blob/46dedfaaa479332159fde74481f74d35215388b5/keyboards/keebio/bfo9000/info.json#L7

though I'm now beginning to realize that this might be a mistake. Although i compiled this board in QMK targeting the Elite-Pi, i don't think this board was actually designed for the Elite-Pi... I think that might be the USB VID for the pro micro...

Reply:
What should I do in this circumstance? I don't see the Elite-Pi listed in the usb-pid repo you linked to

The pre-existing board most similar to the Elite-Pi is the 42. keebs Frood, whose USB PID also isn't listed in the the usb-pid repo

https://github.com/adafruit/circuitpython/blob/05e6db9b092c962a79e9bb3f575b06dd2f71c1b2/ports/raspberrypi/boards/42keebs_frood/mpconfigboard.mk#L2C11-L2C17

thoughts?

Reply:
You can request one from https://pid.codes/

Reply:
Hi @That-Canadian,

I'd love to see official support for The Elite-Pi in CircuitPython, I think it's an awesome board :)

I notice you expressed interest in helping last year in #7141, but the conversation kinda stalled out.

Would you be willing to fill out an application for a USB PID from the raspberry Pi org?

Reply:
Update (quick I know ;) ) : Filled out the application w/ the Raspberry Pi org for a PID, once I hear back will update here and update the VID/PID in the PR.

Reply:
Awesome! From one Canadian to another, Thank you!

Reply:
Hey @alextremblay , got the VID/PID allocation =) , the commit in the raspberry pi repo is here : [PID-Commit](https://github.com/raspberrypi/usb-pid/commit/55ed673dd69c51fd5461b77907cb851308ccf60a)

For clarity, here is the VID/PID allocated for the Elite-Pi : 

**Vendor ID = 0x2E8A**
**Product ID = 0x1072**

You can update your PR or I can update it later this week when I have some free cycles =)

Reply:
Awesome! Thanks a lot!

I've gone ahead and made those changes